examples/qlever_e2e/gantry.env.yml

```diff
@@ -0,0 +1,2 @@
+environment:
+  QLEVER_INDEX: index
```

Comment:
I would add an explicit temporary directory here. This should then also be added as a `.gitignore` empty directory like `index`